src/app/containers/Timestamp/index.jsx

```diff
@@ -9,13 +9,28 @@ const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-
 
 const leadingZero = val => (val < 10 ? `0${val}` : val);
 
-const months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December' ]; // eslint-disable-line
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+]; // eslint-disable-line
 
 // 2019-03-22
 const longNumeric = {
   day: date => leadingZero(date.getDate()),
   month: date => leadingZero(date.getMonth() + 1),
   year: date => date.getFullYear(),
+  hour: date => date.getHours(),
```

Comment:
```suggestion
  hour: date => date.getUTCHours(),
```
this will fix the snapshot tests failing